Utilities/import-llvm

```diff
@@ -15,50 +11,160 @@ import sys
+    'FoldingSet',
+    'Hashing',
+    'IntrusiveRefCntPtr',
+    'None',
+    'Optional',
+    'PointerIntPair',
+    'PointerUnion',
+    'STLExtras',
+    'SmallPtrSet',
+    'SmallSet',
+    'SmallString',
+    'SmallVector',
+    'Statistic',
+    'StringExtras',
+    'StringMap',
+    'StringRef',
+    'StringSet',
+    'StringSwitch',
+    'Triple',
+    'Twine',
+    'edit_distance',
+    'ilist',
+    'ilist_base',
+    'ilist_iterator',
+    'ilist_node',
+    'ilist_node_base',
+    'ilist_node_options',
+    'iterator',
+    'iterator_range',
+    'simple_ilist'
+]
+
+
+
+Demangle_imports = [
+    'Compiler',
+    'ItaniumDemangle',
+    'Utility',
+    'Demangle',
+    'MicrosoftDemangle',
+    'MicrosoftDemangleNodes',
+    'StringView',
+]
 
 # Support types and infrastructure.
 Support_imports = [
-    'AlignOf', 'Allocator', 'Atomic', 'CBindingWrapping', 'Casting', 'Capacity', 'CommandLine', 'Compiler',
-    'Endian', 'Errno', 'ErrorHandling', 'Errc', 'ErrorOr', 'Error', 'Format',
-    'ManagedStatic', 'MathExtras', 'Mutex', 'MutexGuard', 'Memory',
-    'MemoryBuffer', 'PointerLikeTypeTraits', 'Recycler', 'SwapByteOrder',
-    'Timer', 'TimeValue', 'Threading', 'Unicode', 'UniqueLock', 'Unix', 'WindowsError',
-    'Valgrind', 'circular_raw_ostream', 'raw_ostream', 'type_traits', 'JSON']
-
-# Stuff we don't want, but have to pull in.
-Support_imports += [
-    'COFF', 'ConvertUTF', 'ConvertUTFWrapper', 'Debug', 'FileSystem',
-    'FileUtilities', 'Host', 'Locale', 'MachO', 'Path', 'Process', 'Program', 'SMLoc',
-    'SourceMgr', 'Signals', 'StringSaver', 'ToolOutputFile', 'TrailingObjects', 'Unicode', 'UnicodeCharRanges',
-    'MemAlloc', 'Chrono', 'FormatProviders', 'FormatVariadic', 'FormatCommon',
-    'FormatVariadicDetails', 'NativeFormatting', 'DJB', 'ReverseIteration', 'MD5',
-    'SmallVectorMemoryBuffer', 'WithColor', 'Options', 'PrettyStackTrace', 'Watchdog',
-    'TargetParser', 'ARMBuildAttributes', 'ARMTargetParser.def', 'AArch64TargetParser.def', 'X86TargetParser.def', 'LineIterator']
+    'AArch64TargetParser',
+    'APFloat',
+    'ARMBuildAttributes',
+    'ARMTargetParser',
+    'AlignOf',
+    'Allocator',
+    'Atomic',
+    'CBindingWrapping',
+    'Casting',
+    'Chrono',
+    'CommandLine',
+    'Compiler',
+    'ConvertUTF',
+    'ConvertUTFWrapper',
+    'DJB',
+    'DataTypes',
+    'Debug',
+    'Endian',
+    'Errc',
+    'Errno',
+    'Error',
+    'ErrorHandling',
+    'ErrorOr',
+    'FileSystem',
+    'FileUtilities',
+    'Format',
+    'FormatAdapters',
+    'FormatCommon',
+    'FormatProviders',
+    'FormatVariadic',
+    'FormatVariadicDetails',
+    'Host',
+    'LineIterator',
+    'Locale',
+    'MD5',
+    'ManagedStatic',
+    'MathExtras',
+    'MemAlloc',
+    'Memory',
+    'MemoryBuffer',
+    'Mutex',
+    'MutexGuard',
+    'NativeFormatting',
+    'Options',
+    'Path',
+    'PointerLikeTypeTraits',
+    'Process',
+    'Program',
+    'Recycler',
+    'Regex',
```

Comment:
What is pulling in Regex (and all the reg* files below)?  If possible it would be nice to avoid this.

Reply:
Timer -> YAML (so it can print JSON) -> Regex. It's just one function that indexstoredb doesn't use, so it could be commented out/removed.